components/dashboard/src/Login.tsx

```diff
@@ -137,21 +168,37 @@ export function Login() {
                 <div className="flex-grow h-100 flex flex-row items-center justify-center" >
                     <div className="rounded-xl px-10 py-10 mx-auto">
                         <div className="mx-auto pb-8">
-                            <img src={gitpodIcon} className="h-16 mx-auto" alt="Gitpod's logo" />
+                            <img src={providerFromContext ? gitpod : gitpodIcon} className="h-14 mx-auto block dark:hidden" alt="Gitpod's logo" />
+                            <img src={gitpodDark} className="h-14 hidden mx-auto dark:block" alt="Gitpod dark theme logo" />
```

Comment:
Oops, this doesn't look right:
- In light theme, we conditionally switch between Gitpod icon and Gitpod full logo
- But in dark theme, we always use Gitpod full logo, regardless of whether we have `providerFromContext`?

Could be fixed like so:

```suggestion
                            <img src={providerFromContext ? gitpod : gitpodIcon} className="h-14 mx-auto block dark:hidden" alt="Gitpod's logo" />
                            <img src={providerFromContext ? gitpodDark : gitpodIcon} className="h-14 hidden mx-auto dark:block" alt="Gitpod dark theme logo" />
```

Reply:
Thanks 🤦🏻‍♀️

Reply:
I clicked `commit suggestion` but it didn't seem to do it..